src/index.ts

```diff
@@ -24,9 +24,7 @@ export const configureMonacoTailwindcss: typeof import('monaco-tailwindcss').con
 
     return {
       dispose() {
-        while (disposables.length) {
-          disposables.pop()?.dispose();
-        }
+        disposables.map((disposable) => disposable.dispose());
```

Comment:
The difference between these two is that the old version disposes all disposables in reverse order. I’m not sure if it matters.

I prefer using a loop unless the mapped array is actually used.

```suggestion
        for (const disposable of disposables) {
          disposable.dispose();
        }
```

Reply:
```suggestion
        disposables.forEach((disposable) => disposable.dispose());
```

hmm or lets use a simple .forEach -> sure the current code works but its the kindof 'clever' code id like to avoid (eg. dont write clever code, but code that everyone understands)

Reply:
> sure the current code works but its the kindof 'clever' code id like to avoid (eg. dont write clever code, but code that everyone understands)

I agree with that. IMO a `for-of` loop is even simpler, which is why the ESLint rule enforces that. :)